Automattic/jetpack
# CRM: Resolves 2873 - update to moment.js library

Resolves Automattic/zero-bs-crm#2873 - update to moment.js library

## Proposed changes:

JPCRM was using `moment.js` 2.24.0, which was flagged in some systems as having a vulnerability. The security risk really only lies in `npm` server instances:
https://github.com/moment/moment/security/advisories/GHSA-8hfj-j24r-96c4

That said, it's best to avoid even false positives, so I've updated to the latest `moment.js` (with locales), which is `v2.29.4`.

Someday it'd be nice to excise this legacy library, but this is currently used by FullCalendar, and also for legacy datepickers.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






In the `fix/crm/2873-update_moment.js` branch:

1. Go to any JPCRM page, and run `moment.version` in the JS console. It should show `2.29.4`.
2. Go to the task manager calendar. It should load without errors.
3. Make sure the datepicker works when creating a new task.

Everything should work as before.
